src/schema/schema.graphql

```diff
@@ -2010,6 +2010,11 @@ type SMimeMessage {
 	content: String
 }
 
+type createAppointmentResponse {
```

Comment:
instead of creating new type just for two keys, we should use existing type available for appointments (CalendarItemHitInfo)